puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/pipelines.groovy

```diff
@@ -0,0 +1,25 @@
+def installPipelineVars(version, os, type) {
```

Comment:
those two methods only differ in the used playbook and the box-name prefix. I wonder if we can merge them into a `genericPipeline(action, version, os, type)` and have the others as aliases that set action to `install` or `upgrade`?

or will that become too complex if we ever want to differentiate more inside the methods?